connect/runtime/src/test/java/org/apache/kafka/connect/runtime/ErrorHandlingTaskTest.java

```diff
@@ -482,8 +483,8 @@ private void createSourceTask(TargetState initialState, RetryWithToleranceOperat
     }
 
     private ConsumerRecords<byte[], byte[]> records(ConsumerRecord<byte[], byte[]> record) {
-        return new ConsumerRecords<>(Collections.singletonMap(
-                new TopicPartition(record.topic(), record.partition()), singletonList(record)));
+        final TopicPartition tp = new TopicPartition(record.topic(), record.partition());
+        return new ConsumerRecords<>(Collections.singletonMap(tp, singletonList(record)), Collections.singletonMap(tp, new OffsetAndMetadata(record.offset() + 1, record.leaderEpoch(), "")));
```

Comment:
`new ConsumerRecords<L>(Map.of(tp, List.of(record)), Map.of(tp, new OffsetAndMetadata.....` is more concise in Java 11.